Jint/Jint.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <NeutralLanguage>en-US</NeutralLanguage>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
     <AssemblyOriginatorKeyFile>Jint.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Esprima" Version="1.0.0-beta-1036" />
+    <PackageReference Include="VisualOn.Esprima" Version="1.0.0-beta-1036*" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
This should probably have Condition that checks that only if net45

Reply:
And if I remember right ValueTuple can bring a lot of trouble to consuming side prior to 4.7.2 due to versioning, is this really used somewhere...

Reply:
Ah I see, might be a future improvement to remove dependency by using internal struct to pass values inside Dtoa.